app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time = options[:time] ? %(time > "#{options[:time]}") : 'time > yesterday'
```

Comment:
use `>=` instead of `>` due to a scoped_search issue for now.  When you do a greater than search, it actually searches for entries of `time` plus an hour (https://github.com/wvanbergen/scoped_search/blob/v3.2.0/lib/scoped_search/query_builder.rb#L151), so you potentially lose an hour of audits in the report.  `>=` is safer, and there's only an incredibly slim chance of an audit being in two reports, which is better.

The ss behaviour makes sense when testing whole days, but I think it's a bug when you have more precise timestamps, so I'll look at fixing it.